docs/sources/datasources/mssql.md

```diff
@@ -168,122 +168,96 @@ The resulting table panel:
-
-**Example with one `value` and one `metric` column.**
+**Example with `metric` column:**
 
 ```sql
 SELECT
-  time,
-  valueOne,
-  measurement as metric
-FROM
-  metric_values
-WHERE
-  $__timeFilter(time)
+  $__timeGroup(time_date_time, '5m') as time,
+  min("value_double"),
+  'min' as metric
+FROM test_data
+WHERE $__timeFilter(time_date_time)
+GROUP BY $__timeGroup(time_date_time, '5m')
 ORDER BY 1
 ```
 
-When the above query is used in a graph panel, it will produce two series named `Metric A` and `Metric B` with the values `valueOne` and `valueTwo` plotted over `time`.
-
-<div class="clearfix"></div>
-
-{{< figure src="/static/img/docs/v51/mssql_time_series_two.png" class="docs-image--no-shadow docs-image--right" >}}
-
-**Example with multiple `value` columns:**
-
-```sql
-SELECT
-  time,
-  valueOne,
-  valueTwo
-FROM
-  metric_values
-WHERE
-  $__timeFilter(time)
-ORDER BY 1
+Data frame result:
+
+```text
++---------------------+-----------------+
+| Name: time          | Name: min       |
+| Labels:             | Labels:         |
+| Type: []time.Time   | Type: []float64 |
++---------------------+-----------------+
+| 2020-01-02 03:05:00 | 3               |
+| 2020-01-02 03:10:00 | 6               |
++---------------------+-----------------+
 ```
 
-When the above query is used in a graph panel, it will produce two series named `Metric A` and `Metric B` with the values `valueOne` and `valueTwo` plotted over `time`.
-
-<div class="clearfix"></div>
-
-{{< figure src="/static/img/docs/v51/mssql_time_series_three.png" class="docs-image--no-shadow docs-image--right" >}}
-
-**Example using the \$\_\_timeGroup macro:**
+**Example using the fill parameter in the $\_\_timeGroup macro to convert null values to be zero instead:**
 
 ```sql
 SELECT
-  $__timeGroup(time, '3m') as time,
-  measurement as metric,
-  avg(valueOne)
-FROM
-  metric_values
+  $__timeGroup(createdAt, '5m', 0) as time,
+  sum(value) as value,
+  hostname
+FROM test_data
 WHERE
-  $__timeFilter(time)
+  $__timeFilter(createdAt)
 GROUP BY
-  $__timeGroup(time, '3m'),
-  measurement
+  $__timeGroup(createdAt, '5m', 0),
+  hostname
 ORDER BY 1
 ```
 
-When the above query is used in a graph panel, it will produce two series named `Metric A` and `Metric B` with the values `valueOne` and `valueTwo` plotted over `time`.
-Any two series lacking a value in a three-minute window will render a line between those two lines. You'll notice that the graph to the right never goes down to zero.
+Given the data frame result below and using the graph panel you'll get 2 series named _value 10.0.1.1_ and _value 10.0.1.2_ given the name of the field and its labels. If you would like to render the series with a name of _10.0.1.1_ and _10.0.1.2_ you could use a [Standard field options/Display name]({{< relref "../panels/standard-options.md#display-name" >}}) value of `${__field.labels.hostname}`.
```

Comment:
```suggestion
Given the data frame result in the following example and using the graph panel, you will get two series named _value 10.0.1.1_ and _value 10.0.1.2_ given the name of the field and its labels. To render the series with a name of _10.0.1.1_ and _10.0.1.2_ , use a [Standard field options/Display name]({{< relref "../panels/standard-options.md#display-name" >}}) value of `${__field.labels.hostname}`.
```

Reply:
Can we avoid using "given" twice in the first sentence, that will make it an easier read.

Reply:
```suggestion
Given the data frame result in the following example and using the graph panel, you will get two series named _value 10.0.1.1_ and _value 10.0.1.2_. To render the series with a name of _10.0.1.1_ and _10.0.1.2_ , use a [Standard field options/Display name]({{< relref "../panels/standard-options.md#display-name" >}}) value of `${__field.labels.hostname}`.
```